openpype/hosts/flame/plugins/publish/collect_timeline_instances.py

```diff
@@ -126,6 +148,84 @@ def process(self, context):
                 if marker_data.get("reviewTrack") is not None:
                     instance.data["reviewAudio"] = True
 
+    def _get_comment_attributes(self, segment):
+        comment = segment.comment.get_value()
+
+        # try to find attributes
+        attributes = {
+            "xml_overrides": {
+                "pixelRatio": 1.00}
+        }
+        # search for `:`
+        for split in self._split_comments(comment):
+            # make sure we ignore if not `:` in key
+            if ":" not in split:
+                continue
+
+            self._get_xml_preset_attrs(
+                attributes, split)
+
+        # add xml overides resolution to instance data
+        xml_overrides = attributes["xml_overrides"]
+        if xml_overrides.get("width"):
+            attributes.update({
+                "resolutionWidth": xml_overrides["width"],
+                "resolutionHeight": xml_overrides["height"],
+                "pixelAspect": xml_overrides["pixelRatio"]
+            })
+
+        return attributes
+
+    def _get_xml_preset_attrs(self, attributes, split):
+
+        # split to key and value
+        key, value = split.split(":")
+
+        for a_name, a_type in self.xml_preset_attrs_from_comments.items():
+            # exclude all not related attributes
+            if a_name.lower() not in key:
+                continue
+
+            # get pattern defined by type
+            pattern = TXT_PATERN if "string" in a_type else NUM_PATERN
+            res_goup = pattern.findall(value)
+
+            # raise if nothing is found as it is not correctly defined
+            if not res_goup:
+                raise ValueError((
+                    "Value for `{}` attribute is not "
+                    "set correctly: `{}`").format(a_name, split))
+
+            attributes["xml_overrides"][a_name] = res_goup[0]
+
+        # condition for resolution in key
+        if "resolution" in key.lower():
+            res_goup = NUM_PATERN.findall(value)
+            # check if axpect was also defined
+            # 1920x1080x1.5
+            aspect = res_goup[2] if len(res_goup) > 2 else 1
+
+            width = int(res_goup[0])
+            height = int(res_goup[1])
+            pixel_ratio = float(aspect)
+            attributes["xml_overrides"].update({
+                "width": width,
+                "height": height,
+                "pixelRatio": pixel_ratio
+            })
+
+    def _split_comments(self, comment_string):
+        # first split comment by comma
+        split_comments = []
+        if "," in comment_string:
+            split_comments.extend(iter(comment_string.split(",")))
```

Comment:
Why are you adding the `iter` everywhere? `split_comments.extend(comment_string.split(","))` does the job.